com.unity.render-pipelines.universal/Documentation~/post-processing-ssao.md

```diff
@@ -0,0 +1,140 @@
+
+For more information on the Source property, see the section [Implementation details](#implementation-details).
+
+### Normal Quality
+
+This property becomes active when you select the option Depth in the **Source** property.
+
+Higher quality of the normal vectors produces smoother SSAO effect.
+
+Available options:
+
+* **Low**
+* **Medium**
+* **High**
+
+**Performance impact**: medium.
+
+In some scenarios, the **Depth** option produces results comparable with the **Depth Normals** option. But in certain cases, the **Depth Normals** option provides a significant increase in quality. The following images show an example of such case. 
+
+(TODO: screenshots)
+
+For more information, see the section [Implementation details](#implementation-details).
+
+### Intensity
+
+This property defines the intensity of the darkening effect.
+
+**Performance impact**: insignificant.
+
+### Direct Lighting Strength
+
+This property defines how visible the effect is in areas exposed to direct lighting.
+
+(TODO: screenshots)
+
+**Performance impact**: insignificant.
+
+### Radius
+
+When calculating the the Ambient Occlusion value, the SSAO effect takes samples of the normal texture within this radius from the current pixel.
+
+**Performance impact**: high.
+
+Lowering the **Radius** setting improves performance, because the SSAO Renderer Feature samples pixels closer to the source pixel. This makes caching more efficient. 
+
+Calculating the Ambient Occlusion Pass on objects closer to the Camera takes longer than on objects further from the Camera. This is because the **Radius** property is scaled with the object. 
+
+### Sample Count
+
+For each pixel, the SSAO Render Feature takes this number of samples within the specified radius to calculate the Ambient Occlusion value. Increasing this value makes the effect smoother and more detailed, but reduces the performance.
+
+**Performance impact**: high.
+
+Increasing the **Sample Count** value from 4 to 8 doubles the computational load on the GPU.
+
+<a name="#implementation-details"></a>
+
+## Implementation details
+
+The SSAO Renderer Feature uses normal vectors for calculating how exposed each point on a surface is to ambient lighting.
+
+URP 10.0 implements the `DepthNormals` Pass block that generates the the normal texture `_CameraNormalsTexture` for the current frame. By default, the SSAO Renderer Feature uses this texture to calculate Ambient Occlusion values.
+
+If you implement your custom SRP and if you do not want to implement the `DepthNormals` Pass block in your shaders, you can use the SSAO Renderer Feature and set its **Source** property to **Depth**. In this case, Unity does not use the `DepthNormals` Pass to generate the normal, it generates the normal texture using the depth texture instead.
```

Comment:
_"...it generates the normal texture using the depth texture instead."_

Change it to:
_"...it reconstructs the normals from the depth texture instead."_